app/assets/stylesheets/new_design/card_admin.scss

```
@@ -14,6 +14,10 @@
   margin-bottom: 20px;
   margin-right: 20px;
 
+  .badge.baseline {
```

Pinaillage : je dirais même `.badge` tout court : après tout on va sans doute vouloir donner de la marge au badge quoi qu'il arrive, baseline ou pas, non ?

```suggestion
  .badge {
```